presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +using a hash of the join key. This can be slower       +
++                                          +                                          +                                    +(sometimes substantially) than broadcast joins but     +
++                                          +                                          +                                    +allows much larger joins. Broadcast joins require that +
++                                          +                                          +                                    +the tables on the right side of the join fit in memory +
++                                          +                                          +                                    +on each machine whereas distributed joins only need to +
++                                          +                                          +                                    +fit in distributed memory across all machines. This    +
++                                          +                                          +                                    +can also be specified on a per-query basis using the   +
++                                          +                                          +                                    +``distributed_join`` session property.                 +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++   ``node-scheduler.network-topology``    +   ``String`` (``legacy`` or ``flat``)    +             ``legacy``             +Sets the network topology to use when scheduling       +
++                                          +                                          +                                    +splits. ``legacy`` will ignore the topology when       +
++                                          +                                          +                                    +scheduling splits. ``flat`` will try to schedule       +
++                                          +                                          +                                    +splits on the same host as the data is located by      +
++                                          +                                          +                                    +reserving 50% of the work queue for local splits.      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.default-concurrency``       +               ``Integer``                +               ``1``                +Default local concurrency for parallel operators. Use  +
```

Comment:
Is default of "1" reasonable in light of our recent findings? Should we suggest a bigger value, maybe one that scales with the number of CPU cores? Or offer any other guideline?